src/Services/OData/ODataFormatter.cs

```diff
@@ -274,7 +274,7 @@ internal void WriteContentProperty(String path, string propertyName, bool rawVal
 
             if (propertyName == ODataHandler.PROPERTY_ACTIONS)
             {
-                WriteActionsProperty(portalContext, ODataTools.GetActionItems(content, req).ToArray(), rawValue);
+                WriteActionsProperty(portalContext, ODataTools.GetHtmlActionItems(content, req).ToArray(), rawValue);

```

Comment:
Please do not roll back this as this fix was the main reason we started this thread: to include all actions in the list, not just html ones.